tests/test_constraints_polars.py

```diff
@@ -109,3 +111,43 @@ def test_polars_exclusion(mock_substances, parameters, constraints):
     df = ldf.filter((pl.col("Pressure") < 3) & (pl.col("Temperature") > 120)).collect()
     num_entries = len(df)
     assert num_entries == 0
+
+
+@pytest.mark.parametrize("parameter_names", [["Solvent_1", "Solvent_2", "Solvent_3"]])
+@pytest.mark.parametrize("constraint_names", [["Constraint_7"]])
+def test_polars_label_duplicates(parameters, constraints):
+    """Tests Polars implementation of no-label duplicates constraint."""
+    ldf = _lazyframe_from_product(parameters)
+
+    ldf = _apply_polars_constraint_filter(ldf, constraints)
+
+    ldf = ldf.with_columns(
+        n_unique=pl.concat_list(pl.col(["Solvent_1", "Solvent_2", "Solvent_3"]))
```

Comment:
do you have access to `parameter_names` here? if so lets use it instead of the hardcoded list, if not then forget it

Reply:
No, I didn't have access.

Reply:
Saw this to late, but just for your information: you can simply add it as additional argument to the function and it will be automatically made available by the fixture